bin/cml/runner/launch.js

```diff
@@ -76,6 +76,26 @@ const shutdown = async (opts) => {
     }
   };
 
+  const destroyLeo = async () => {
+    if (!tfResource) return;
+
+    winston.info(`Waiting ${destroyDelay} seconds to destroy`);
+    await sleep(destroyDelay);
+
+    const { cloud, id, name, region } = JSON.parse(
+      Buffer.from(tfResource, 'base64').toString('utf-8')
+    ).instances[0].attributes;
+
+    try {
+      // FIXME: a crafted name can result in arbitrary code execution
+      return await exec(
+        `leo destroy-runner --cloud="${cloud}" --name="${name}" --region="${region}" ${id}`
+      );
```

Comment:
* https://github.com/iterative/terraform-provider-iterative/pull/695
```suggestion
      return await exec(
        `leo destroy-runner --cloud=${cloud} --region=${region} ${id}`
      );
```
Still would be safer to escape properly or use [`child_process.execFile`](https://nodejs.org/api/child_process.html#child_processexecfilefile-args-options-callback) but we control these values... more or less. 🙃